rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -21,6 +20,9 @@ object SystemProcesses {
 
   private val prettyPrinter = PrettyPrinter()
 
+  // because only we do installs
+  private implicit val INSTALLS_MATCH_PHLOS = matchListQuote(CostAccount(Integer.MAX_VALUE))
```

Comment:
I don't like this idea because it will appear multiple times in the file and it's safe to have it implicit in the `SystemProcesses`.

Reply:
I insist :) Won't be able to sleep at night otherwise :joy:

Reply:
I've had a look at the whole file and I don't see a single install. I see a couple different usages of rspace that don't seem to have a different implicit in scope... Looks like a bug?

Reply:
Shouldn't the end-user pay for invoking system contracts? If I get it right, it's for free ATM.

Reply:
Yes, at the moment using the crypto functions is free. `INSTALLS_MATCH_PHLOS` is a limit used in RSpace  to perform a match between data and patterns on the channel. The channel is a return/`ack` channel given by the user. It will most likely be an unforgeable one (and soon it will be only forgeable) so there should be only one `consume`/`for` listening on it. The data is single element - either `boolean` or `hash`. The match will be easy one.